src/technology-consulting/3-prioritization.md

```diff
@@ -19,28 +19,27 @@ prioritize what should be done.
 
 Developing a viable product is one of the most difficult challenges
 anyone will undertake. However, there are some processes that
-__must__ be followed to achieve success.
+**must** be followed to achieve success.
 
+**No matter what stage of the product delivery cycle you are in, you should:**
 
-No matter what stage of the product delivery cycle you are in, you should:
+<input type="checkbox"/> Have clearly articulated and measurable [learn-technology-consulting/goals goals].
 
-- <input type="checkbox"/> Have clearly articulated and measurable [learn-technology-consulting/goals goals].
-- <input type="checkbox"/> Document what needs to be done in a prioritized backlog. ([Backlogs](#backlog)).
+<input type="checkbox"/> Document what needs to be done in a prioritized backlog. ([Backlogs](#backlogs)).
```

Comment:
Extra period not needed, might make sense to just move (Backlogs) before the period

Reply:
Fixed! Good catch